inspire/dojson/institutions/schemas/insitutions-0.0.1.json

```diff
@@ -129,11 +168,6 @@
             "type": "array",
             "uniqueItems": true
         },
-        "non_public_note": {
-            "description": "FIXME: what is the difference with hidden_note!?",
-            "title": "Non public note",
-            "type": "string"
-        },
```

Comment:
This was corresponding to a different MARC field that the hidden note. Have you matched both to `hidden_note` in dojson?